date-time/README.md

```diff
@@ -38,16 +38,23 @@ timer.elapsed().collect { elapsed: Duration ->
     // simulate some work
     delay(100)
 }
+println("Done!")
 ```
+
 The timer implementation makes the best effort to compensate the inaccuracies in intervals. 
 In addition, one or several frames will be skipped in case the consumer processing takes longer than
-[interval]. So the output might look the following way:
+[interval]. 
+A running timer automatically changes to a [Finished] state once [duration] elapses.
+[Finished] is a terminal state and following [start] calls would have no effect.
+
+So the output might look the following way:
```

Comment:
will the last value always be the max now?

Reply:
yes. actually it always was. 
in the current simplified solution it might be possible that there would be 2 outputs in the elapsed sequence close together (like `...59995, 60000`, or even `...60000, 60000`)- one from the `Running` state and another from `Finished`

Reply:
but never `60001` right? I'd just add this to the README.

Reply:
done